pkg/skaffold/deploy/docker/deploy.go

```diff
@@ -265,7 +265,14 @@ func (d *Deployer) Dependencies() ([]string, error) {
 	return nil, nil
 }
 
-func (d *Deployer) Cleanup(ctx context.Context, out io.Writer) error {
+func (d *Deployer) Cleanup(ctx context.Context, out io.Writer, dryRun bool) error {
+	// if dryRun {
+	// 	output.Yellow.Fprintln(out, "Following containers would be deleted:")
+	// 	for _, container := range d.tracker.DeployedContainers() {
+	// 		output.Yellow.Fprintf(out, "* %s", container.ID)
+	// 	}
+	// 	return nil
+	// }
```

Comment:
This is the right thing to do. Is there a reason why this is commented out?

Reply:
I've uncommented the code yet when running the `docker-deploy` example I'm not able to make it print any container.ID. For some reason I don't know, `d.tracker.DeployedContainers()` returns an empty map

Reply:
@kadern0 is this after doing a skaffold run? I think currently `skaffold delete` on the docker deployer [isn't implemented properly](https://github.com/GoogleContainerTools/skaffold/issues/6314), but will soon in the future.

So running `skaffold run` then `skaffold delete` won't yield results because `skaffold delete` won't properly track the containers that were spun up

Reply:
That was my suspicion but didn't look at the code. Thanks for the info!